src/pages/iou/IOUModal.js

```diff
@@ -325,34 +344,48 @@ class IOUModal extends Component {
                             {didScreenTransitionEnd && (
                                 <>
                                     {currentStep === Steps.IOUAmount && (
-                                        <IOUAmountPage
-                                            onStepComplete={(amount) => {
-                                                this.setState({amount});
-                                                this.navigateToNextStep();
-                                            }}
-                                            reportID={reportID}
-                                            hasMultipleParticipants={this.props.hasMultipleParticipants}
-                                            selectedAmount={this.state.amount}
-                                            navigation={this.props.navigation}
-                                        />
+                                        <AnimatedStep
+                                            direction={this.getDirection()}
+                                            style={[styles.flex1, styles.pageWrapper]}
+                                        >
+                                            <IOUAmountPage
+                                                onStepComplete={(amount) => {
+                                                    this.setState({amount});
+                                                    this.navigateToNextStep();
+                                                }}
+                                                reportID={reportID}
+                                                hasMultipleParticipants={this.props.hasMultipleParticipants}
+                                                selectedAmount={this.state.amount}
+                                                navigation={this.props.navigation}
+                                            />
+                                        </AnimatedStep>
                                     )}
                                     {currentStep === Steps.IOUParticipants && (
-                                        <IOUParticipantsPage
-                                            participants={this.state.participants}
-                                            hasMultipleParticipants={this.props.hasMultipleParticipants}
-                                            onAddParticipants={this.addParticipants}
-                                            onStepComplete={this.navigateToNextStep}
-                                        />
+                                        <AnimatedStep
+                                            style={[styles.flex1]}
+                                            direction={this.getDirection()}
+                                        >
```

Comment:
Is there a reason that each of the AnimatedStep has a different style?

Reply:
I just moved the style from the `View` that was wrapping our steps (like in the previous comment) to the `AnimatedStep`, that's why the style is different for each step. 

This was necessary because wrapping the step with a `AnimatedStep` (`Animatable.View`) without a style was changing the styling of the step.

Reply:
Can't we leave flex1 on the `AnimatedStep` and put `pageWrapper` style within this component?

Reply:
Hmm. I think it's bad to have the container style spread through multiple views, but yes I can change it!

Reply:
Hmm, let's not do it then. It's just not obvious why the steps have different styles here and I worry this is confusing.